packages/cli/test/unit/clone-example.unit.js

```diff
@@ -7,7 +7,7 @@
 
 const promisify = require('util').promisify;
 
-const cloneExampleFromGitHub = require('../generators/example/clone-example');
+const cloneExampleFromGitHub = require('../../generators/example/clone-example');
```

Comment:
This file contains an integration test, please move it to `packages/cli/test/integration/clone-example.integration.js`.

IIRC, most other tests in this repository are end-to-end, therefore they should go to `acceptance` directory.

Reply:
> IIRC, most other tests in this repository are end-to-end, therefore they should go to acceptance directory.

I see that your pull request moved those files to `integration` directory, that's a good solution too. I am never entirely sure what's the distinction between integration and acceptance categories.

Reply:
For the CLI package, my intention was to have everything test whether `npm test` or `npm start` work on the generated apps/extensions live in `acceptance` while everything else would live in `integration` or `unit`. On second look, I agree that this file should stay with the other 'generator' tests